pkg/ebpf/c/tracee.bpf.c

```diff
@@ -763,6 +765,15 @@ struct new_module_map {
 
 typedef struct new_module_map new_module_map_t;
 
+struct module_context_map {
+    __uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+    __uint(max_entries, 3);
+    __type(key, u32);
+    __type(value, u64);
+} module_context_map SEC(".maps");
```

Comment:
It will be more efficient to define the map value to be a struct that contains node addr, idx, and iteration num.
By doing that, only one map lookup will be required to access all three, and less code will be required both for getting and setting these values